packages/cursorless-engine/src/processTargets/TargetPipelineRunner.ts

```diff
@@ -347,17 +347,14 @@ function targetsToVerticalTarget(
       anchorTarget.contentRange.end.character,
     );
 
-    if (anchorTarget instanceof PositionTarget) {
-      results.push(anchorTarget.withContentRange(contentRange));
-    } else {
-      results.push(
-        new PlainTarget({
-          editor: anchorTarget.editor,
-          isReversed: anchorTarget.isReversed,
-          contentRange,
-        }),
-      );
-    }
+    results.push(
+      new VerticalRangeTarget({
```

Comment:
How come you need a new target type here?

Reply:
It behaves like a plan target with no special removal range with the exception of the insertion delimiter being given as an argument. We had no existing target like that.

Reply:
Just didn't want to make the definition of plain target looser. If you prefer that go ahead.